### PR TITLE
Update project configuration

### PR DESCRIPTION
**Pull request by jaakkonakaza**
# Description of the PR
Based on the IntelliJ Platform Plugin Template
https://github.com/JetBrains/intellij-platform-plugin-template/tree/ijpgp-2.0-EAP

Github Actions will be updated later

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        password = environment("PRIVATE_KEY_PASSWORD")
+    }
+
+    publishing {
+        token = environment("INTELLIJ_PUBLISH_TOKEN")
+        // The pluginVersion is based on the SemVer (https://semver.org) and supports pre-release labels,
+        // like 2.1.7-alpha.3
+        // Specify pre-release label to publish the plugin in a custom Release Channel automatically.
+        // Read more:
+        // https://plugins.jetbrains.com/docs/intellij/deployment.html#specifying-a-release-channel
```
Comment (OlliKiljunen) [inline build.gradle.kts:134]: This seems to be a bit out of place here.

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </codeStyleSettings>
+    <codeStyleSettings language="vgo">
+      <indentOptions>
+        <option name="USE_TAB_CHARACTER" value="false" />
+      </indentOptions>
+    </codeStyleSettings>
+  </code_scheme>
+</component>
```
Comment (OlliKiljunen) [inline .idea/codeStyles/Project.xml:82]: If we add some code style things to repo, I wish there was some check for missing newlines in the EOF. :D

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -43,7 +30,7 @@
                text="A+"
                description="Actions for A+ menu"
                popup="true">
-            <add-to-group group-id="MainMenu" anchor="last"/>
+<!--            <add-to-group group-id="MainMenu" anchor="last"/>-->
```
Comment (OlliKiljunen) [inline src/main/resources/META-INF/plugin.xml:33]: Are we losing the menu with this PR?

**Review by OlliKiljunen — COMMENTED** [review]
Make sure the menu does not disappear yet; it's still needed in the master branch.

And the CI build is failing. Can you say, why?

**Comment by OlliKiljunen** [comment]
>And the CI build is failing. Can you say, why?

Okay, perhaps because

> Github Actions will be updated later

Anyway, I'd like to see those updates included to this PR before merging, so we can see the CI building.

**Comment by jaakkonakaza** [comment]
> Make sure the menu does not disappear yet; it's still needed in the master branch.

Note that this will be merged to the 4.0 branch, which will be merged to master when everything is completed

**Review by OlliKiljunen — APPROVED** [review]
> Note that this will be merged to the 4.0 branch, which will be merged to master when everything is completed

Oh, you're right. :P

**Pull request merged**